unlock-app/src/pages/creator/index.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
```

Comment:
Previously `creator/LockMaker.js`, not sure if we should decouple this into a separate component if we would like to reuse it and load it into the page.